schema.sql

```diff
@@ -24,3 +24,30 @@ create table species(
   id integer generated always as identity primary key,
   name varchar
 );
+
+/* Create a table named vets */
+create table vets(
+  id integer generated always as identity primary key,
+  name varchar,
+  age integer,
+  date_of_graduation date
+);
+
+/* Create table specializations many to many relationship between species and vets */
+create table specializations(
+  species_id integer,
+  vets_id integer,
+  primary key(species_id, vets_id),
+  constraint fk_species foreign key(species_id) references species(id) on delete cascade,
+  constraint fk_vets foreign key(species_id) references vets(id) on delete cascade
+);
+
+/* Create table visits many to many relationship between animals and vets */
+create table visits(
+  animals_id integer,
+  vets_id integer,
+  date_of_visit date,
+  primary key(animals_id, vets_id),
```

Comment:
- [ Optional ] It would be better to remove this constraint on the `visits` table since one `animal` can make many visits to the same `vet`. Placing a primary key on the `animals_id` and `vets_id` would make this impossible. I highly recommend you do this.